internal/mill/exec.go

```diff
@@ -54,6 +55,28 @@ func (p *Petsitter) run(t *skylark.Thread, fn *skylark.Builtin, args skylark.Tup
 	return skylark.None, nil
 }
 
+func (p *Petsitter) start(t *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
+	var cmdV skylark.Value
+
+	if err := skylark.UnpackArgs("cmdV", args, kwargs,
+		"cmdV", &cmdV,
+	); err != nil {
+		return nil, err
+	}
+
+	cmdArgs, err := argToCmd(fn, cmdV)
+	if err != nil {
+		return nil, err
+	}
+
+	cwd, _ := os.Getwd()
+	if _, err := p.Runner.StartWithIO(cmdArgs, cwd, p.Stdout, p.Stderr); err != nil {
+		return nil, err
+	}
+
+	return skylark.None, nil
```

Comment:
one thing that we haven't spec'd out is that 'start' needs to have some sort of return value that lets you do things with the running process. but i'm not sure yet what this should do!

a simple place to start would be to have this return a skylark.Dict with the Pid in a field?
https://godoc.org/github.com/google/skylark#Dict